tests/tests-connect.js

```diff
@@ -0,0 +1,65 @@
+import test from 'ava'
+import sleep from 'sleep-promise'
+
+import sbClient, { SpacebroClient } from '../src/spacebro-client'
+
+test('Simple connect', async (t) => {
+  sbClient.connect('spacebro.space', 3333, {
```

Comment:
do also the tests with instancing objects `new SpacebroClient('spacebro.space', 3333, ...`